### PR TITLE
Add ADOT workshop

### PR DESCRIPTION
**Pull request by ivyostosh**
*Description of changes:* Add ADOT workshop

*Testing (if applicable):* Locally deployed. See sample below

Screen Shot 2022-09-30 at 7 42 38 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/workshops/packages/adot/adot_amp_amg.md [hunk 1]
```diff
@@ -0,0 +1,607 @@
...
+1. the Prometheus Remote Write Exporter, which employs the remote write features and send metrics to AMG for long term storage;
+
+1. the Sigv4 Authentication Extension, which enables ADOT pods to authenticate to AWS services.
+
+Follow steps below to complete the ADOT package installation:
+
+1. Update the following config file. Review comments carefully and replace everything that is wrapped with a `<>` tag. Note this configuration aims to mimic the Prometheus community helm chart. A user can tailor the scrap targets further by modifying the receiver section below. Refer to [ADOT package spec]({{< relref "../../../reference/packagespec/adot" >}}) for additional explanation of each section.
+
+    <details>
+      <summary>Click to expand ADOT package config</summary>
```
Comment (ivyostosh) [inline docs/content/en/docs/workshops/packages/adot/adot_amp_amg.md:125]: This section by default will be collapsed

File: docs/content/en/docs/workshops/packages/adot/adot_amp_amg.md [hunk 2]
```diff
@@ -0,0 +1,607 @@
...
+                  processors: [batch/metrics]
+                  exporters: [logging, prometheusremotewrite]
+
+      ```
+    </details>
+
+1. Bind additional roles to the service account `pod-identity-webhook` created at step [Create an EKS-A cluster with IRSA](), as `pod-identity-webhook` by design does not have sufficient permissions to scrap all Kubernetes targets above. 
+
+    <details>
+      <summary>Click to expand clusterrole and clusterrolebinding config</summary>
```
Comment (ivyostosh) [inline docs/content/en/docs/workshops/packages/adot/adot_amp_amg.md:453]: This section by default will be collapsed

**Review by vincentni — APPROVED** [review]
/lgtm

**Pull request merged**